### DISCUSSION TITLE
dockge's  yaml parser is choking on a docker compose that works normally

### DISCUSSION THREAD

**Discussion by mrelcee** [louislam/dockge#633] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

a docker compose yaml for firing up a unifi controller with the unifi app and mongodb in one compose fails to parse and flags an error in dockge.     runs fine however with docker compose.   

 when I peeked at it in the stacks folder it's intact there, so it isn't getting mangled in the webui.

I believe the issue is over a pipe |  character toward the end..

[sample.zip](https://github.com/user-attachments/files/17349081/sample.zip)

### 🐻 Dockge Version

1.42

### 💻 Operating System and Arch

Arch linux on x64 and  Debian on arm64

### 🌐 Browser

safari chrome firefox..

### 🐋 Docker Version

Docker version 27.3.1, build ce1223035a

### 🟩 NodeJS Version

not using

**Comment by trythatagain** [comment]
Not sure if this is going to help, but I when I changed to the new unifi application, I did use the linuxserver.io setup, but I've just looked and it slightly different to how I did it, I did fix mongo to version **7** and had no problems.
I also don't set my ports up like you have either. I set mine up like this

```
    ports:
      - 8443:8443
      - 3478:3478/udp
      - 10001:10001/udp
      - 8080:8080
    #  - 1900:1900/udp #optional
    #  - 8843:8843 #optional
    #  - 8880:8880 #optional
    #  - 6789:6789 #optional
    #  - 5514:5514/udp #optional
```

In fact have you tried just following the linuxserver.io write up?
